src/app/catalog/containers/equipment-item/equipment-item.component.html

```diff
@@ -1,53 +1,186 @@
-<section class="equipment-item" >
-  <img #image class="equipment-item__img" [src]="defaultImage" alt="equipment photo">
+<section class="equipment-item">
+  <img
+    #image
+    class="equipment-item__img"
+    [src]="defaultImage"
+    alt="equipment photo"
+  />
   <h2 class="equipment-item__title">{{ equipment?.title }}</h2>
   <div class="equipment-item__description">
     {{ equipment?.description }}
   </div>
   <div class="equipment-info-list">
     <div class="equipment-info-list__item">
       <div class="info-item">
-        <h3 class="info-item__title">Срок</h3>
-        <button
+        <!-- <h3 class="info-item__title">Срок</h3> -->
+        <div
           mat-icon-button
           (click)="tooltip.show()"
-          mat-raised-button #tooltip="matTooltip"
+          mat-raised-button
+          #tooltip="matTooltip"
           matTooltip="Максимальный срок использования оборудования, дней"
-          matTooltipPosition="right">
-          <mat-icon>help_outline</mat-icon>
-        </button>
-        <div>До {{ equipment?.maximumDays}} суток</div>
+          matTooltipPosition="right"
+        >
+          <mat-icon
+            ><svg
+              width="23"
+              height="23"
+              viewBox="0 0 23 23"
+              fill="none"
+              xmlns="http://www.w3.org/2000/svg"
+            >
+              <path
+                d="M2.03056 2.50378V6.29165H5.81844M1.0835 11.5C1.0835 17.2529 5.74719 21.9166 11.5002 21.9166C17.2531 21.9166 21.9168 17.2529 21.9168 11.5C21.9168 5.74701 17.2531 1.08331 11.5002 1.08331C7.64495 1.08331 4.27886 3.17764 2.47765 6.29066M11.5028 5.24998L11.5021 11.5046L15.9185 15.9209"
+                stroke="#4D8F9E"
+                stroke-width="1.5"
+                stroke-linecap="round"
+                stroke-linejoin="round"
+              />
+            </svg>
+          </mat-icon>
+        </div>
```

Comment:
Do you try to find this icon in materials icons?

Reply:
I didn't try to find any icons in materials icons, because Aleksei Popov said that I can use svg. I'll try to find it if needed.

Reply:
Done. I found it in materials icons.